Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+
+def  build_non_monolithic_code_modules( project_path: pathlib.Path,
+                                        non_mono_build_path: pathlib.Path):
+    #build the pre_requisite tools if needed
+    os.makedirs(non_mono_build_path, exist_ok=True)
+
+    process_command(["cmake", "-B", non_mono_build_path, "-S", project_path])
+
+    process_command(["cmake", "--build", non_mono_build_path, "--target", "AssetProcessorBatch", "AssetBundlerBatch", "--config", "profile", "--", "/m"])
+
+def build_monolithic_code_modules( project_path: pathlib.Path,
+                        project_name: str,
+                        build_config: str,
+                        mono_build_path: pathlib.Path,
+                        should_build_game_launcher: bool,
+                        should_build_server_launcher: bool,
+                        should_build_unified_launcher: bool):
+    process_command(["cmake", "-B", mono_build_path, "-S", project_path, "-DLY_MONOLITHIC_GAME=1", "-DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0"])
+
+    os.makedirs(mono_build_path, exist_ok=True)
+    if should_build_server_launcher:
+
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.ServerLauncher", "--config", build_config, "--", "/m"])
+
+    if should_build_game_launcher:
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.GameLauncher", "--config", build_config, "--", "/m"])
+    
+    if should_build_unified_launcher:
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.UnifiedLauncher", "--config", build_config, "--", "/m"])
+
+
+def kill_existing_processes(project_name: str):
+    o3de_process_names = ['o3de', 'editor', 'assetprocessor', f'{project_name.lower()}.serverlauncher', f'{project_name.lower()}.gamelauncher' ]
+    for process in psutil.process_iter():
+        processes_to_kill = []
+        # strip off .exe and check process name
+        if os.path.splitext(process.name())[0].lower() in o3de_process_names:
+            processes_to_kill.append(process)
+        
+        if processes_to_kill:
+            logging.error("The following processes are still running: "+[process.name() for process in processes_to_kill])
+            user_input = input(f'Continuing may cause build errors.\nStop them before continuing? (y/n). Quit(q)')
+            if user_input.lower() == 'y':
+                for p in processes_to_kill:
+                    p.terminate()
+                    p.wait()
+            elif user_input.lower() == 'q':
+                quit()
+
+
+def process_assets( project_name: str,
+                    project_path: pathlib.Path,
+                    non_mono_build_path: pathlib.Path):
+    process_command(["cmake", "-B", non_mono_build_path, "-S", project_path])
+
+    process_command(["cmake", "--build", non_mono_build_path, "--target", "AssetProcessorBatch", "AssetBundlerBatch", "--config", "profile", "--", "/m"])
```

Comment:
Can't this be done by calling `build_non_monolithic_code_modules` instead?

Reply:
good catch! Adding that in now.